udata/core/followers/factories.py

```diff
@@ -0,0 +1,12 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import unicode_literals
+
+from factory.mongoengine import MongoEngineFactory
+
+from udata import models
+
+
+class FollowFactory(MongoEngineFactory):
```

Comment:
In the tests, it is nicer to call factories. Otherwise, I have to call factories and then, for some other objects, I have to call a constructor and .save().

But it is up to you. Do you want me to suppress these factories ?